src/renderer/viz/expressions/basic/constant.js

```diff
@@ -29,17 +29,24 @@ export default class Constant extends BaseExpression {
         checkNumber('constant', 'x', 0, x);
 
         super({});
-        this.expr = x;
+        this.value = x;
```

Comment:
Are we missing here the underscore?
```suggestion
        this._value = x;
```